pom.xml

```diff
@@ -514,6 +507,60 @@
                 <version>${jackson.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.fasterxml.jackson.dataformat</groupId>
+                <artifactId>jackson-dataformat-yaml</artifactId>
+                <version>${jackson.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-core</artifactId>
+                <version>1.6.1</version>
```

Comment:
Why not `swagger.version`?

Reply:
Yeah, I removed it and put it back fighting enforcer. Maybe I don't need it now. 
I will replace with version if it is still needed. 
On the other hand, versions through properties prevent maven auto update versions via command line, but that's not a good enough reason